packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -71,6 +71,9 @@ const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
  */
 const generateImage = (image, alt, isAriaHidden, variant, handleFallback) => {
   const icon = generateImagePlaceholder(alt, isAriaHidden, variant);
+  if (variant === 'user') {
```

Comment:
This could be simplified, I think. 

```
const imageFit = (variant === 'user') ? ImageFitTypes.COVER : undefined;
return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} imageFit={imageFit} />
```

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/0127dec949032aa00c86bb77ebdeb92d3d695832.